modules/signing_worker/manifests/init.pp

```diff
@@ -126,23 +224,16 @@
         purge   => true,
         force   => true,
     }
-
-    python::virtualenv { "signingworker_${user}" :
-        ensure          => present,
-        version         => '3',
-        requirements    => $tmp_requirements,
-        venv_dir        => $virtualenv_dir,
-        ensure_venv_dir => true,
-        owner           => $user,
-        group           => $group,
-        timeout         => 0,
-        # This is not strictly necessary, but if Puppet ever runs
-        # from a cwd that the signing worker user can't access,
-        # we end up hitting this pip bug:
-        # https://github.com/pypa/pip/issues/9445
-        cwd             => $scriptworker_base,
-        path            => [ '/bin', '/usr/bin', '/usr/sbin', '/usr/local/bin', '/Library/Frameworks/Python.framework/Versions/3.8/bin'],
+    exec { "install ${scriptworker_base} widevine":
+        command     => "${virtualenv_dir}/bin/python setup.py install",
+        cwd         => $widevine_clone_dir,
+        user        => $user,
+        group       => $group,
+        refreshonly => true,
+        subscribe   => [Exec["clone widevine ${scriptworker_base}"], Python::Virtualenv["signingworker_${user}"]],
+        require     => [Exec["clone widevine ${scriptworker_base}"], Python::Virtualenv["signingworker_${user}"]],
     }
+
     # XXX once we:
     #     - get the virtualenv to re-run pip on requirements.txt change,
     #     - get the scriptworker and poller to restart on config or python
```

Comment:
It looks like we still need this at some point, yes? Essentially, this PR will install a new version of scriptworker onto a running machine, but the old version of scriptworker will remain running until we ssh in and manually kick it. Accurate?

(I'm ok with that as a follow-up; just making sure we track it if it's still needed work.)

Reply:
Yes, that's right. I don't think it's safe to fix this part until we have secrets managed by Puppet.